main.go

```diff
@@ -145,6 +146,7 @@ func main() {
 	pflag.BoolVar(&enableGoInstrumentation, constants.FlagGo, false, "Controls whether the operator supports Go auto-instrumentation")
 	pflag.BoolVar(&enablePythonInstrumentation, constants.FlagPython, true, "Controls whether the operator supports python auto-instrumentation")
 	pflag.BoolVar(&enableNginxInstrumentation, constants.FlagNginx, false, "Controls whether the operator supports nginx auto-instrumentation")
+	pflag.BoolVar(&enableNodeJSInstrumentation, constants.FlagNodeJS, false, "Controls whether the operator supports nodejs auto-instrumentation")
```

Comment:
The flat is in Beta, which means it should be enabled by default.

You should also actually delete the flag.